service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/azure/container/CopyAzureStorageContainerDefinitionStep.java

```diff
@@ -79,27 +75,24 @@ public StepResult doStep(FlightContext flightContext)
     var destinationResourceId =
         inputParameters.get(
             WorkspaceFlightMapKeys.ControlledResourceKeys.DESTINATION_RESOURCE_ID, UUID.class);
-    var destStorageAccountId =
-        flightContext
-            .getWorkingMap()
-            .get(
-                WorkspaceFlightMapKeys.ControlledResourceKeys
-                    .DESTINATION_STORAGE_ACCOUNT_RESOURCE_ID,
-                UUID.class);
+
+    var destStorageAccountInfo =
```

Comment:
This is now the storage account object that contains either the landing zone resource ID or the storage account ID.